app/views/propertyLinking/uploadEvidence.scala.html

```diff
@@ -132,59 +132,58 @@ <h2 class="govuk-error-summary__title" id="error-summary-heading">@messages("err
+                        @errors.map { error =>
+                            <p id="file-upload-1-error" class="govuk-error-message">
                                 <span class="govuk-visually-hidden">@messages("accessibility.error.label")</span> @messages(error)
-                            }
-                            </span>
+                            </p>
                         }
-                        <div class="govuk-form-group">
-                            <label class="govuk-label--s" for="newFileButton">@messages("common.uploadFile.label")
-                               <div id="fileTypesText" class="govuk-hint">
-                                   @messages("uploadOtherEvidence.uploadFile.hint")
-                               </div>
-                            </label>
-                            <input id="newFileButton" class="govuk-file-upload" name="file-upload-1" type="file">
-                        </div>
-                    </div>
-                }else{
-                    @govukWarningText(WarningText(
-                        iconFallbackText = messages("warning.text"),
-                        content = Text(messages("businessRatesAttachment.file.size.reached.max.limit"))
-                    ))
-                }
+                    }
+                    <input id="newFileButton" class="govuk-file-upload" name="file-upload-1" type="file">
+                </div>
+            </div>
+        }
 
-                <div class="message-warning govuk-!-display-none" id="message-warning"><p class="govuk-body">@messages("fileupload.pleaseWait")</p></div>
-
-               @if(uploadedFiles.nonEmpty) {
-                   <table class="govuk-table">
-                       <tbody class="govuk-table__body">
-                       @uploadedFiles.map { case (fileReference, upload) =>
-                       <tr class="govuk-table__row">
-                           <td class="govuk-table__cell">@{
-                               upload.fileMetadata.toDisplayFileName
-                           }</td>
-                           <td class="govuk-table__cell">
-                               <button id="remove-file" type="submit" class="govuk-button govuk-button--secondary" formaction="@UploadController.remove(fileReference, EvidenceChoices.OTHER)">
-                               @messages("remove.file")
-                       </button>
-                       </td>
-                       </tr>
-                       }
-                       </tbody>
-                   </table>
-               }
+        <div class="message-warning govuk-!-display-none" id="message-warning"><p class="govuk-body">@messages("fileUpload.pleaseWait")</p></div>
+
+        @if(uploadedFiles.nonEmpty) {
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("fileUpload.descriptionList.key")
```

Comment:
could you switch this to use the injected `messages("fileUpload.descriptionList.key")` instead?